src/path_converters.h

```diff
@@ -363,10 +366,30 @@ class PathClipper : public EmbeddedQueue<3>
                     goto exit_loop;
 
                 case agg::path_cmd_move_to:
+
+                    // was the last command a moveto (and we have
+                    // seen at least one command ?
+                    // if so, shove it in the queue if in clip box
+                    if (m_moveto && m_has_init &&
+                        m_lastX >= m_cliprect.x1 &&
+                        m_lastX <= m_cliprect.x2 &&
+                        m_lastY >= m_cliprect.y1 &&
+                        m_lastY <= m_cliprect.y2) {
+                        // push the last moveto onto the queue
```

Comment:
I wrote this as comments to my self while understanding this.  I do not see a problem with leaving them.